sdk/dashboard/Azure.ResourceManager.Dashboard/api/Azure.ResourceManager.Dashboard.netstandard2.0.cs

```diff
@@ -0,0 +1,146 @@
+namespace Azure.ResourceManager.Dashboard
+{
+    public static partial class ArmClientExtensions
+    {
+        public static Azure.ResourceManager.Dashboard.GrafanaResource GetGrafanaResource(this Azure.ResourceManager.ArmClient client, Azure.Core.ResourceIdentifier id) { throw null; }
+    }
+    public partial class GrafanaResource : Azure.ResourceManager.Core.ArmResource
```

Comment:
A resource usually should not end with `Resource`

Reply:
I will ask service to make the change from swagger